src/formatDate.js

```diff
@@ -9,6 +9,62 @@
  */
 function formatDate(date, fromFormat, toFormat) {
   // write code here
+  const dateObj = parseData(date, fromFormat);
+
+  return getData(dateObj, toFormat);
 }
 
+function parseData(date, fromFormat) {
+  const dateObj = {};
+
+  for (let i = 0; i < 3; i++) {
+    const part = date.split(fromFormat[3])[i];
```

Comment:
The date.split(fromFormat[3]) is called in each iteration of the loop, which is unnecessary. You should perform the split operation once before the loop and use the resulting array inside the loop.